src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -51,15 +51,54 @@ public boolean connect(String database, String user, String password) {
 
     private int getTableRowsCount(String tableName) throws SQLException {
         try (Statement statement = connection.createStatement();
-             ResultSet resultSet = statement.executeQuery(TABLE_ROWS_COUNT_QUERY.replace("?",tableName))) {
+             ResultSet resultSet = statement.executeQuery(TABLE_ROWS_COUNT_QUERY.replace("?", tableName))) {
             resultSet.next();
             return resultSet.getInt(1);
         }
     }
 
-    public void closeConnection() {
+    public DataSet[] getTableData(String tableName) {
+        String query = String.format("SELECT * FROM \"%s\"", tableName);
+        if ((connection != null) && (hasTable(tableName))) {
+            try (Statement statement = connection.createStatement();
+                 ResultSet resultSet = statement.executeQuery(query)) {
+                if (!resultSet.isBeforeFirst()) {
+                    return new DataSet[0];
+                }
+                int tableRowsCount = getTableRowsCount(tableName);
+                DataSet[] tableData = new DataSet[tableRowsCount];
+                int columnCount = resultSet.getMetaData().getColumnCount();
+                for (int rowIndex = 0; resultSet.next(); rowIndex++) {
+                    tableData[rowIndex] = new DataSet(columnCount);
+                    for (int columnIndex = 0; columnIndex < columnCount; columnIndex++) {
+                        tableData[rowIndex].add(columnIndex, resultSet.getString(columnIndex + 1));
+                    }
+                }
+                return tableData;
+            } catch (SQLException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+        return new DataSet[0];
+    }
+
+    private boolean hasTable(String tableName) {
+        String query = String.format("SELECT to_regclass('%s')", tableName);
+        try (Statement statement = connection.createStatement();
+             ResultSet resultSet = statement.executeQuery(query)) {
+            resultSet.next();
+            return resultSet.getString(1) != null;
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
```

Comment:
Why you are handling exceptions this way? This is not a handling, just printing `e.getMessage()` to console. There is some approcah "Throw early, catch late", here you can find more explanations - http://bit.ly/2Bo9zQP. It's better to rethrow SQLException in this case. But if you still would like to log somehow ( like sout() ) the exception message you can do it this way:
```
} catch (SQLException e) {
    System.out.println(e.getMessage());
    throw e;
}
```